Add content to Docker page for review. (#114)
input-output-hk/cardanodocs.com: _docs/cardano-node/node-running-docker.md

parent: running-a-cardano-node
order: 2
---
Read if: You want to use Docker to run a Cardano node

Read first: [Running a Cardano node](../cardano-node.md)

> Read next: [Configuring a Cardano node using a config.yaml file](node-config-yaml.md)

Reviewer: `cardano-node` developers will be the best contacts for assistance with this page. The Docker image simply defines a mount point for the user to pass configuration in at runtime if not wishing to use the IOHK-managed config.

> You can use Docker to quickly set up and run a Cardano node. Multiple Docker images are available, each preconfigured to connect to a specific public Cardano network. There is also a base image available, which runs an unconfigured Cardano node that can be provided with a custom `config.yaml` file. All of the preconfigured Docker images contain this base image with an additional configuration layer.

Reviewer: > Multiple Docker images are available, each preconfigured to connect to a specific public Cardano network

_Tags suitable for development and production dependency pinning are pushed to the Dockerhub registry, with default configuration for joining public supported networks available by setting the `NETWORK` environment variable at runtime_

> There is also a base image available, which runs an unconfigured Cardano node that can be provided with a custom `config.yaml` file

There's been a late change to the design that condenses the base and network variants to a single image. At runtime the user can elect to use the IOHK-managed configuration for each public, or omit the NETWORK and simply mount config into the container.

> For a full list of the available Docker images, see the official [Cardano node Docker Hub](link to Cardano node docker Hub) page. <!-- Do we need to define the PIN format here too?-->

Reviewer: ```suggestion
See the IOHK [Cardano node Docker Hub repository](https://hub.docker.com/repository/docker/inputoutput/cardano-node) for the available tags.
```

## Prerequisites

> You need to have Docker installed and configured on your machine. See the [Docker documentation](https://docs.docker.com/install/) for more information.

Reviewer: This can be refined when the use-cases are separated

> ## Steps

Reviewer: I can also refine this later, as it's dependent on the context
